src/diepvries/driving_key_field.py

```diff
@@ -31,3 +37,33 @@ def __str__(self) -> str:
             f"{type(self).__name__}: field_name={self.name}, "
             f"satellite={self.satellite_name}"
         )
+
+    def _validate(self):
+        """Validate driving key.
+
+        Perform the following checks:
+        1. Satellite name has a prefix within TABLE_PREFIXES[TableType.SATELLITE].
+        2. Parent table name has a prefix within TABLE_PREFIXES[TableType.LINK].
+
+        Raises:
+            RuntimeError: If one of the checks fails.
+        """
+        satellite_name_prefix = next(
+            split_part for split_part in self.satellite_name.split("_")
+        )
+        if satellite_name_prefix not in TABLE_PREFIXES[TableType.SATELLITE]:
+            raise RuntimeError(
```

Comment:
I'm not sure if `RuntimeError` is the best exception here. Would like to hear @matthieucan 's opinion on it. I think it would be better to raise something like an `InputError` or so, no?

Ideally we'd raise a specific `diepvries` exception like `InvalidSatelliteNameError`, but given that we did not really create custom exceptions in `diepvries`, maybe it would be better to do a larger alignment and raise custom exceptions in other places as well, which might be an overkills for the scope of this PR.

Reply:
Those are good points, and indeed we should have more custom exceptions everywhere consistently.
For this PR, a possibility could be to run an `assert satellite_name_prefix in...` and let Python raise an `AssertionError`? WDYT?

Reply:
The exception message can also be customized by doing `assert condition, message`

Reply:
I like it! Nice work.